suite-native/module-device-settings/src/components/DeviceAutoConnectCard.tsx

```diff
@@ -1,15 +1,40 @@
 import { useCallback } from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 
+import { useNavigation } from '@react-navigation/native';
+import { isRejected } from '@reduxjs/toolkit';
+
+import { removeThpAutoconnectThunk } from '@suite-common/thp';
 import { selectDeviceAutoconnectCredentials } from '@suite-common/wallet-core';
+import { useAlert } from '@suite-native/alerts';
 import { CompactCardWithIconLayout } from '@suite-native/atoms';
-import { useTranslate } from '@suite-native/intl';
-import { useThpAutoconnectAlerts } from '@suite-native/thp';
+import { Translation, useTranslate } from '@suite-native/intl';
+import {
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { useThpAutoconnectActions } from '@suite-native/thp';
+import { useToast } from '@suite-native/toasts';
+import TrezorConnect from '@trezor/connect';
+
+type NavigationProp = StackNavigationProps<
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes.DeviceSettings
+>;
 
 export const DeviceAutoConnectCard = () => {
     const { translate } = useTranslate();
-    const { showThpAutoconnectTurnOffAlert, showThpAutoconnectTurnOnAlert } =
-        useThpAutoconnectAlerts();
+
+    const { showAlert } = useAlert();
+
+    const { showToast } = useToast();
+
+    const dispatch = useDispatch();
+
+    const navigation = useNavigation<NavigationProp>();
+
+    const { startAutoconnect } = useThpAutoconnectActions();
```

Comment:
Too many blanks lines for me... 😅 What about this?

```suggestion
    const { translate } = useTranslate();
    const { showAlert } = useAlert();
    const { showToast } = useToast();

    const dispatch = useDispatch();
    const navigation = useNavigation<NavigationProp>();

    const { startAutoconnect } = useThpAutoconnectActions();
```

Reply:
Went for this: 
<img width="807" height="250" alt="image" src="https://github.com/user-attachments/assets/43325539-f3ef-473e-83b1-07468032f2bf" />